### ISSUE TITLE
Just to explore the option, pls ignore

### ISSUE THREAD

**Issue by rajukoyagurasnow** [ckallakuri/ckallakuri.github.io#1]
@ckallakuri pls ignore this :)

Did you get any notification?

Here I attached a file, can you access it?
[Agile Guided Setup.pdf](https://github.com/ckallakuri/ckallakuri.github.io/files/4932572/Agile.Guided.Setup.pdf)

**Comment by rajukoyagurasnow** [comment]
Hmm..we can use each issue as a story here.

Even we can directly paste screen shots, this is cool 👍 
![image](https://user-images.githubusercontent.com/68386482/87692912-ee526880-c7a9-11ea-9bca-008f7b78b15a.png)

**Comment by ckallakuri** [comment]
@rajukoyagurasnow Yes Raju 👍 This looks good. Will explore more. May need to your help to understand some of these when we meet next. Thanks.
